<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWidget</class>
 <widget class="QWidget" name="AuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(35, 35, 35);
}</string>
  </property>
  <widget class="QPushButton" name="signin_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(34, 34, 34);
    text-decoration: none;
    /* margin-top: 10px; */
    padding: 1px;
    text-transform: None;
    transition: 0.5s;
    display: inline-block;
    position: relative;
	background-color:rgb(169, 255, 186);
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	border-radius: 20px;
}

QPushButton:hover {
	border: rgb(255, 170, 0);
    /* transform: translateY(3px);*/
    /*transform: skewY(-1deg); */
    cursor: pointer;
    background-color: rgb(115, 255, 148);
}</string>
   </property>
   <property name="text">
    <string>sign in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(34, 34, 34);
    text-decoration: none;
    /* margin-top: 10px; */
    padding: 1px;
    text-transform: None;
    transition: 0.5s;
    display: inline-block;
    position: relative;
	background-color:rgb(169, 255, 186);
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	border-radius: 20px;
}

QPushButton:hover {
	border: rgb(255, 170, 0);
    /* transform: translateY(3px);*/
    /*transform: skewY(-1deg); */
    cursor: pointer;
    background-color: rgb(115, 255, 148);
}</string>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passcode_lineedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	padding-left: 1em;
	color: white;
	background-color: rgb(104, 104, 104);
	border-radius: 20px;
	border:2px solid  rgb(104, 104, 104) ;
}

QLineEdit:hover {
	color: rgb(178, 246, 159);
    cursor: pointer;
    background-color: rgba(104, 104, 104, 0.911);
}

QLineEdit:focus {
	border-color:rgb(169, 255, 186);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>passcode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_lineedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	padding-left: 1em;
	color: white;
	background-color: rgb(104, 104, 104);
	border-radius: 20px;
	border:2px solid  rgb(104, 104, 104) ;
}

QLineEdit:hover {
	color: rgb(178, 246, 159);
    cursor: pointer;
    background-color: rgba(104, 104, 104, 0.911);
}

QLineEdit:focus {
	border-color:rgb(169, 255, 186);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>login</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_lable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	padding-left: 1em;
	color: white;
	background-color: rgb(104, 104, 104);
	border-radius: 20px;
	border:2px solid  rgb(104, 104, 104) ;
	text-transform: uppercase;
}

/*QLabel:hover {
	color: rgb(178, 246, 159);
    cursor: pointer;
    background-color: rgba(104, 104, 104, 0.911);
}*/
</string>
   </property>
   <property name="text">
    <string>sign in or register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
